app/models/author.rb

```diff
@@ -17,22 +17,9 @@ def alternative_identities
   end
 
   # Provide consistent API for Author and AuthorIdentity
-  # @return [String] preferred_first_name
-  def first_name
-    preferred_first_name
-  end
-
-  # Provide consistent API for Author and AuthorIdentity
-  # @return [String] preferred_middle_name
-  def middle_name
-    preferred_middle_name
-  end
-
-  # Provide consistent API for Author and AuthorIdentity
-  # @return [String] preferred_last_name
-  def last_name
-    preferred_last_name
-  end
+  alias_attribute :first_name, :preferred_first_name
+  alias_attribute :middle_name, :preferred_middle_name
+  alias_attribute :last_name, :preferred_last_name
```

Comment:
This seems to lose the yard docs, can they be retained?

Reply:
They are no longer necessary, because the knowledge about these is the same as the knowledge about those ActiveRecord attributes.  You know these are strings because the schema defines them as as strings.  Also, repeating `Provide consistent API for Author and AuthorIdentity` is not informative.  What value do think is lacking?